### PR TITLE
Fix/token seal

### PR DESCRIPTION
**Pull request by pilunte23**

### CODE REVIEW

**Comment by pilunte23** [comment]
fix this issue https://github.com/GeckoTH/arkham-horror/issues/231

**Comment by pilunte23** [comment]
I load subtype information in shared pile now
it 's fix for https://github.com/GeckoTH/arkham-horror/issues/212

File: o8g/Scripts/actions.py [hunk 1]
```diff
@@ -861,7 +870,7 @@ def addBlessCurse(group, isBless, who=me):
 
     #check current Tokens in Bag
     updateBlessCurse()
-    if ((cb.markers[Bless] >= 10) and isBless) or ((cb.markers[Curse] >= 10) and not isBless):
+    if ((cb.markers[Bless] + b >= 10) and isBless) or ((cb.markers[Curse] + c >= 10) and not isBless):
```
Comment (ChristopherS) [inline o8g/Scripts/actions.py:873]: The idea is, that the numbers on the chaosbag represent the total number of tokens in play. Therefore this shouldn't be necessary.

File: o8g/Scripts/actions.py [hunk 2]
```diff
@@ -842,11 +842,20 @@ def updateBlessCurse():
...
-        return
-
+        return  
+  
+    # Search Seal curse and bless token
+    for card in table:
+        if card.name == "Bless" and card.Subtype == "Sealed":
+            b += 1
+        if card.name == "Curse" and card.Subtype == "Sealed":    
+            c += 1
```
Comment (ChristopherS) [inline o8g/Scripts/actions.py:858]: At first I thought this shouldn't be needed, but I now realize, that there are a few edge cases, where the number of sealed Blurse Tokens should also be counted. Still this should be part of updateBlessCurse() and the total added to the displayed value on the chaosbag.

File: o8g/Scripts/actions.py [hunk 3]
```diff
@@ -1810,6 +1819,7 @@ def sealTokenCard(card, x = 0, y = 0, player = None):
     card.Subtype = 'Sealed'
     card.filter = "#99999999"
     card.controller = player
+    updateBlessCurse()
```
Comment (ChristopherS) [inline o8g/Scripts/actions.py:1822]: The idea, why I didn't add this here was, that sealing a Blurse-Token, doesn't change the number of Blurse-tokens in play. So this update is unnecessary.

File: o8g/Scripts/actions.py [hunk 4]
```diff
@@ -1837,6 +1847,7 @@ def sealToken(group, x = 0, y = 0, player = None):
...
```
Comment (ChristopherS) [inline o8g/Scripts/actions.py:1850]: Same for this.

**Review by ChristopherS — CHANGES_REQUESTED** [review]
Just some clarification how the chaosbag display is supposed to work.

**Review by ChristopherS — CHANGES_REQUESTED** [review]
Just some clarification how the chaosbag display is supposed to work.

**Pull request merged**